packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -441,7 +442,13 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
             },
             this._onRenderContainer
           )}
-        <div role="region" aria-live="polite" aria-atomic="true" id={errorMessageId} className={this._classNames.errorMessage}>
+        <div
+          role="region"
+          aria-live="polite"
+          aria-atomic="true"
+          id={errorMessageId}
+          className={hasText ? this._classNames.errorMessage : ''}
```

Comment:
This seems to hinge adding a class name based on whether or not the combobox itself has text; is that the intent? It seems like an error when the combo is empty is a legit case.

Reply:
Yes this is the intent - right now it's adding a margin even when there's no text making it look like the div has empty space.